src/Core/Addon/Module/ModuleRepository.php

```diff
@@ -109,29 +111,43 @@ class ModuleRepository implements ModuleRepositoryInterface
      */
     private $cacheProvider;
 
+    /**
+     * @var ConfigurationInterface
+     */
+    private $configuration;
+
     /**
      * Keep loaded modules in cache.
      *
      * @var DoctrineProvider
      */
     private $loadedModules;
 
+    /**
+     * @var AdapterModuleRepository
+     */
+    private $moduleRepository;
```

Comment:
isn't it weird to have an attribute $moduleRepository in the ModuleRepository class. Maybe rename it to $adapterModuleRepository